staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1/types.go

```diff
@@ -52,12 +52,20 @@ type ExecCredentialSpec struct {
 }
 
 // ExecCredentialStatus holds credentials for the transport to use.
+//
+// Token and ClientKeyData are sensitive fields. This data should only be
+// transmitted in-memory between client and exec plugin process. Exec plugin
+// itself should at least be protected via file permissions.
 type ExecCredentialStatus struct {
 	// ExpirationTimestamp indicates a time when the provided credentials expire.
 	// +optional
 	ExpirationTimestamp *metav1.Time `json:"expirationTimestamp,omitempty"`
 	// Token is a bearer token used by the client for request authentication.
 	Token string `json:"token,omitempty"`
+	// PEM-encoded client TLS certificate (without CA data).
```

Comment:
in some cases, intermediate CA data might be required. no reason to require this to omit that... we should handle a blob containing 1 or more certificate PEM blocks just like golang, which takes the first as the client cert and the rest as the intermediate pool to use

Reply:
Done